packages/arb-token-bridge-ui/src/components/TransferPanel/AdvancedSettings.tsx

```diff
@@ -119,7 +120,11 @@ const AdvancedSettings = ({
               className="w-full"
               // we want to keep the input empty for the same wallet address
               // placeholder only displays it to the user for assurance
-              placeholder={!isSmartContractWallet ? walletAddress : undefined}
+              placeholder={
+                !isSmartContractWallet && walletAddress
+                  ? shortenAddress(walletAddress)
```

Comment:
hm i think showing the unshortened version is better because the input should be a full, unshortened address
the placeholder should reflect the format of the actual input

Reply:
Removed the placeholder for now